dist/api_data.py

```diff
@@ -1906,8 +1819,7 @@ def __ge__(self, other):
         modified. If true, this option forces the checkpoint''',
         type='boolean'),
     Config('name', '', r'''
-        if set, specify a name for the checkpoint (note that checkpoints including LSM trees
-        may not be named)'''),
+        if set, specify a name for the checkpoint (note that checkpoints may not be named)'''),
```

Comment:
For my curiosity: are there any other types of trees that may not be named?

Reply:
Thanks for pointing this one out. It looks like no it was just for LSM trees. [This](https://github.com/wiredtiger/wiredtiger/pull/550) is the commit that introduced the check. I will reword remove this note from the config description, it no longer makes sense.

Reply:
Fixed https://github.com/wiredtiger/wiredtiger/pull/11273/commits/9757a68c4c4b1ad5780d2d1461bedf1d35c650f5